_integration-schemas/closeio/v1/activities.md

```diff
@@ -0,0 +1,347 @@
+    - name: _type
```

Comment:
The top part of the table file should also be in here. [Here's an example using Amplitude](https://github.com/stitchdata/docs/pull/138/files#diff-297b85aa367660bb5cf774953567342b).

Also note that the content of these files must be enclosed in a pair of `---`, which are on their own lines:

```
---
[file content]
...
...
---
```

Otherwise, Jekyll won't read and render the files.